charts/latest-v2/azuredisk-csi-driver/values.yaml

```diff
@@ -135,6 +135,19 @@ controller:
       requests:
         cpu: 10m
         memory: 20Mi
+  environments:
```

Comment:
Are the values under environments used anywhere?

Reply:
This was from an early iteration. Removed.